ghub-graphql.el

```diff
@@ -45,8 +44,8 @@ behave as for `ghub-request' (which see)."
   (cl-assert (stringp graphql))
   (cl-assert (not (stringp variables)))
   (ghub-request "POST" "/graphql" nil :payload
-                (json-encode `(("query" . ,graphql)
```

Comment:
@tarsius I have a question on this function, do we really need to encode this to json here? I thought the payload will be converted to json eventually via `ghub--encode-payload` later

Reply:
No it doesn't need to. I've changed that.